ONSdigital/eq-survey-runner
# Runner to use CDN

How to test

Requires thorough manual regression testing by a few people. All variations of the user interface need to be tested. Any deviation in changes to the following media types need to be considered when testing:
- Confirm all script behaviour is correct and in line with behaviour seen in the pattern library.
- Confirm all css styles are correct and in line with the pattern library.
- Confirm all font types are still working.
- Confirm all images are referenced correctly from CSS and HTML.
- Confirm all existing EQ themes work.
- Confirm all favicons are referenced correctly.
